content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+window.
+
+At the top of the pop-up window is the product image, name, and price. Beneath that are any
+available :ref:`extra items or options <lunch/extra>`. Click the checkbox next to any extras to add
+them to the order.
+
+Each extra is organized by a category, and lists the name and price for each extra. As extras are
+selected, the displayed price at the top of the pop-up window updates to reflect all current
+selections.
+
+Beneath the extras is the :guilabel:`Description` of the product, followed by a :guilabel:`Notes`
+field. The :guilabel:`Notes` field is used to enter any information to be sent to the vendor
+regarding the order, such as any special requests or food allergies.
+
+When all selections for the product have been made, click the :guilabel:`Add To Cart` button in the
+lower-left of the pop-up window. To cancel the order, click the :guilabel:`Discard` button.
+
+.. image:: orders/pizza.png
+   :align: center
+   :alt: The pop-up window for a personal pizza, with all the extras highlighted and selected.
+
+Errors
+~~~~~~
+
+Depending on how the various :ref:`extras <lunch/extra>` are configured for a vendor, it is possible
+to receive an error when attempting to add products to the cart.
+
+An error can occur when a product is configured to require a selection be made for an extra, but no
+selection is made for that extra category.
+
+When this occurs, a :guilabel:`Validation Error` pop-up window appears. The error is briefly
+explained in the pop-up window. Click :guilabel:`Close` to close the window, and make any necessary
+changes to the :guilabel:`Configure Your Order` pop-up window.
+
+.. example::
+   The vendor, The Pizza Palace, provides a free beverage with any purchase. Their products are
+   configured so that a beverage selection is required when adding one of their products to the
+   cart.
+
+   If a selection is **not** made, an error occurs. The message that appears is `You have to
+   order one and only one Free Beverage with Purchase`.
+
+   .. image:: orders/error.png
+      :align: center
+      :alt: The :guilabel:`Validation Error` pop-up window with the specific error for the free
+            beverage displayed.
+
+Your Order summary
+------------------
+
+At the top of the dashboard in the :guilabel:`Your Order` summary, the account information appears
+as well as all the information relating to all orders placed for the day.
+
+As products are added to an order, they appear at the top center of the summary box. Each product is
+listed beneath the words :guilabel:`Your Order`, with the product name, quantity, and a status tag.
+
+The available tags that can be displayed for each item are:
+
+- :guilabel:`To Order`: the product has been added to the cart, but has not been purchased yet.
+- :guilabel:`Ordered`: the product has been purchased by the user and is waiting to be sent to the
+  vendor.
+- :guilabel:`Sent`: the product has been sent to the vendor.
```

Comment:
What determines this? Scheduled action? Every so many minutes? How long will customer have to wait?